docker-compose.yml

```diff
@@ -297,7 +297,7 @@ services:
     volumes:
       - ./:/repo
   oathkeeper:
-    image: oryd/oathkeeper:v0.40.1
+    image: oryd/oathkeeper:v0.40.2
```

Comment:
confirmed that this brings the oathkeeper version to the same as used in charts: https://github.com/ory/k8s/tree/master/helm/charts/oathkeeper